scanner/providers/implementations/themoviedatabase.py

```diff
@@ -210,40 +274,27 @@ async def for_language(lng: str) -> Movie:
 				tagline=movie["tagline"] if movie["tagline"] else None,
 				tags=list(map(lambda x: x["name"], movie["keywords"]["keywords"])),
 				overview=movie["overview"],
-				posters=self.get_image(
-					movie["images"]["posters"]
-					+ (
-						[{"file_path": movie["poster_path"]}]
```

Comment:
do we not need the `poster_path` and things like that from the base api? I added them because it always provided a good fallback if the `images` list was empty.